pkg/parser/validate/jobs_test.go

```diff
@@ -315,7 +315,7 @@ func TestJobTypeValidation(t *testing.T) {
 					End:   protocol.Position{Line: 2, Character: 18},
 				},
 				Severity: protocol.DiagnosticSeverityError,
-				Message:  "Invalid job type 'bad-type'. Allowed types: approval, build, no-op, release",
+				Message:  "Invalid job type 'bad-type'. Allowed types: approval, build, lock, no-op, release, unlock",
```

Comment:
I realise this list is in alphabetical order, but since lock and unlock are paired I wonder if it makes sense to group them tighter? How about "approval, build, lock & unlock, no-op, release"

Reply:
Agreed! Makes more sense when reading it out.

Reply:
The error is built from just printing out the string slice from `utils/jobs.go`. I'll place `lock` and `unlock` next to each other so the list is nicer